src/redux/books/books.js

```diff
@@ -1,25 +0,0 @@
-// eslint-disable-next-line
```

Comment:
**You did a great job! Your styling matches the main requirement!**

However, we have some cleaning jobs to do! Let me show you the places that need to be cleaned.

1. Kindly remove this code from `BookForm.css`

<img width="676" alt="Screenshot 2023-04-06 at 17 58 20" src="https://user-images.githubusercontent.com/69011963/230386856-f5005acc-2694-4cd4-8192-eaa5608808de.png">

2. Kindly remove eslint disabler from `BookForm.js`

<img width="678" alt="Screenshot 2023-04-06 at 17 58 51" src="https://user-images.githubusercontent.com/69011963/230387047-67b13c3e-931b-414a-804d-0debf986adec.png">

2. Kindly remove eslint disabler from `booksSlice.js`

<img width="678" alt="Screenshot 2023-04-06 at 17 59 12" src="https://user-images.githubusercontent.com/69011963/230387140-4bff4068-f05c-4347-9ab9-7691a1148232.png">

> 🔧 Sometimes we may encounter challenges when using ESLint. Unexpected errors may arise which can be difficult to resolve quickly. If you come across an issue that is functioning properly but is still displaying an error message when uploaded to GitHub, I recommend disabling it in the .eslintrc.json file. This can help to prevent any confusion or delays in the development process. Let me know if you need any assistance.